src/store/sagas.js

```diff
@@ -192,6 +193,29 @@ function* updateStatusVal(scope, action) {
   }
 }
 
+/**
+ * The search function to be called with parameters when the search is hit
+ * @param {*} scope
+ * @param {*} action
+ */
+function* searchNgoData(scope, action) {
```

Comment:
You can reuse the existing search function instead of writing a new one.
`yield takeLatest(ngoReportTypes.SEARCH, search, ngoReportTypes, Api.searchNgoForm);`

Reply:
Done. Thanks for the suggestion.